10up/ElasticPress
# Add Did you Mean in Instant Results

### Description of the Change

This PR adds the suggestion from the Did You Mean feature in the Instant Results 


Closes #3409

### How to test the Change

1) Enable Did You Mean and Instant Results
2) Create a post with the same content and title, "10 Up Loves WordPress."
3) Go to the frontend and search for" wordpre."
4) The Instant Results modal should show the suggestion "WordPress"


### Changelog Entry

> Added - Integrate Did You Mean feature in the Instant Results


### Credits

Props @burhandodhy 


### Checklist:
